content/docs/use-cases/sharing-resources-efficiently.md

```diff
@@ -0,0 +1,79 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use og special
+processors, and minimize data transfers. This involves provisioning and managing
+resources such as massive on-prem data stores and powerful servers, which can be
+expensive and time consuming.
+
+![](/img/shared-server.png) _Data store shared by DVC projects_
+
+DVC projects support different ways to optimize resource utilization in order to
+minimize cost and complexity. This can make the difference, for example when:
+
+- Multiple users work on the same shared server, or there's a single computing
+  environment to run experiments.
+- GPU time gets distributed among people or processes for training machine
+  learning models.
+- There's a centralized data storage unit or cluster.
+
+## Shared Storage
+
+Individual DVC projects already use a local data <abbr>cache</abbr> to achieve
+near-instantaneous <abbr>workspace</abbr> restoration when switching among
+[versions of data](/doc/use-cases/versioning-data-and-model-files), results,
+etc. (think **Git for data**).
+
+The cache directory is fully customizable (see `dvc config cache`), including
+it's location, so nothing prevents you from having it in a
+[location shared](/doc/user-guide/how-to/share-a-dvc-cache) by multiple local
+copies of a <abbr>project</abbr>, or even by different projects altogether. This
+enables DVC's automatic de-duplication of data files across all projects.
+
+Additionally, optional [remote storage](/doc/command-reference/remote) e.g.
```

Comment:
Somewhere here we should get back to the initial problem we stated and explain how does this solution help. Also solution itself -we should jump a bit higher level and simplify it further.

Reply:
> get back to the initial problem we stated and explain how does this solution help
solution itself -we should jump a bit higher level

I added some clarifications both for local cache and remote storage. Not sure if that makes it higher level, I guess it's still a bit specific (referring to the 2 basic optimizations built into the data cache).